manifests/profile/www_lib/vhosts/apps_lib.pp

```diff
@@ -192,7 +192,7 @@
       },
       {
         rewrite_cond => '%{REQUEST_URI} !^/openid-connect',
```

Comment:
This condition can be removed now. It's probably also best to add a comment about what we are redirecting to (a standalone drupal-based SALI.

Reply:
Removed and comment added.